crates/biome_css_analyze/src/keywords.rs

```diff
@@ -5433,17 +5433,293 @@ pub const RESET_TO_INITIAL_PROPERTIES_BY_FONT: [&str; 13] = [
     "font-variation-settings",
 ];
 
+// https://developer.mozilla.org/ja/docs/Web/HTML/Element
+// https://github.com/sindresorhus/html-tags/blob/main/html-tags.json
+pub(crate) const HTML_TAGS: [&str; 148] = [
+    "a",
+    "abbr",
+    "acronym",
+    "address",
+    "applet",
+    "area",
+    "article",
+    "aside",
+    "audio",
+    "b",
+    "base",
+    "basefont",
+    "bdi",
+    "bdo",
+    "bgsound",
+    "big",
+    "blink",
+    "blockquote",
+    "body",
+    "br",
+    "button",
+    "canvas",
+    "caption",
+    "center",
+    "cite",
+    "code",
+    "col",
+    "colgroup",
+    "content",
+    "data",
+    "datalist",
+    "dd",
+    "del",
+    "details",
+    "dfn",
+    "dir",
+    "div",
+    "dl",
+    "dt",
+    "em",
+    "embed",
+    "fencedframe",
+    "fieldset",
+    "figcaption",
+    "figure",
+    "font",
+    "footer",
+    "form",
+    "frame",
+    "frameset",
+    "h1",
+    "h2",
+    "h3",
+    "h4",
+    "h5",
+    "h6",
+    "head",
+    "header",
+    "hgroup",
+    "hr",
+    "html",
+    "i",
+    "iframe",
+    "img",
+    "input",
+    "ins",
+    "isindex",
+    "kbd",
+    "keygen",
+    "label",
+    "legend",
+    "li",
+    "link",
+    "listbox",
+    "listing",
+    "main",
+    "map",
+    "mark",
+    "marquee",
+    "math",
+    "menu",
+    "menuitem",
+    "meta",
+    "meter",
+    "model",
+    "multicol",
+    "nav",
+    "nextid",
+    "nobr",
+    "noembed",
+    "noframes",
+    "noscript",
+    "object",
+    "ol",
+    "optgroup",
+    "option",
+    "output",
+    "p",
+    "param",
+    "picture",
+    "plaintext",
+    "portal",
+    "pre",
+    "progress",
+    "q",
+    "rb",
+    "rp",
+    "rt",
+    "rtc",
+    "ruby",
+    "s",
+    "samp",
+    "script",
```

Comment:
`shadow` is missing.

<hr />

Here's some browser history if you are curious:

- Firefox 29 added support for `<shadow>` behind the `dom.webcomponents.enabled` flag in late 2013.
- The tag was removed from the specification in 2015.

ref
https://bugzilla.mozilla.org/show_bug.cgi?id=887538
https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/29
https://web.archive.org/web/20150429040731/http://w3c.github.io/webcomponents/spec/shadow/#the-shadow-element